src/inet/IPAddress.h

```diff
@@ -386,6 +388,20 @@ class DLL_EXPORT IPAddress
      */
     static bool FromString(const char * str, size_t strLen, IPAddress & output);
 
+    /**
+     * @brief
+     *   Scan the IP address from its conventional presentation text, including
+     *   the interface ID if present. (e.g. "fe80::2%wlan0").
+     *
+     * @param[in]    str          A pointer to the text to be scanned.
+     * @param[out]   addrOutput   The object to set to the IP address.
+     * @param[out]   ifaceOutput  The object to set to the interface ID.
```

Comment:
Need to document what happens if there is no interface id in the string.

Reply:
Done